packages/cli/src/__tests__/e2e/build.spec.ts

```diff
@@ -229,6 +229,25 @@ describe("e2e tests for build command", () => {
   })
 
   describe("Local strategy", () => {
+
+    // Local strategy runs `yarn` by default, so we need to ensure that we clean up lockfiles
```

Comment:
I'm not sure local strategy should run yarn by default

Reply:
It's pretty normal for build scripts to run the package install command once before building a project (angular does this by default, so do .net projects).

If you feel differently I'd certainly want to open an issue in which we could discuss this.

Reply:
I don't see any cons TBH even if the user has already installed all the deps because it won't install things if it already exists. However, I don't have a specific preference because installing deps yourself is common before building.

Reply:
Depends on the build process. Yarn doesn't do it
I think we could discuss this, but as part of another issue